src/grpc/grpc_server.cc

```diff
@@ -2388,9 +2407,15 @@ Server::Server(
   model_stream_infer_cq_ = builder_.AddCompletionQueue();
 
   // A common Handler for other non-inference requests
```

Comment:
Move this comment to just above L2415.
and add a new comment saying here that response_delay is only used for testing and debugging purposes.